spec/integration_spec.cr

```diff
@@ -0,0 +1,48 @@
+require "./spec_helper"
+
+include ShouldRunSuccessfully
+include WithProjectCleanup
+include ModifyFiles
+
+describe "Initializing a new web project with Breeze" do
+  it "runs a full integration" do
+    puts "Breeze Specs: Running integration spec. This might take awhile...".colorize(:yellow)
```

Comment:
I find that always putting to STDOUT in specs can be annoying when piping or dont other terminal commands what do you thing about using a logger here at the default log level. 

```suggestion
    Log.notice { "Breeze Specs: Running integration spec. This might take awhile...".colorize(:yellow) }
```

```
2021-05-17T18:44:43.237189Z NOTICE - Breeze Specs: Running integration spec. This might take awhile...
```

his can create more noise in the console but I find it to be more usable.

Reply:
This is actually for the person developing Breeze so they see a little notice. It's pulled from here https://github.com/luckyframework/lucky_cli/blob/master/spec/integration/init_web_spec.cr#L7

Reply:
true but even as a developer it is nice to not have forced output to the console.  I guess it is just a habit I have after some frustrations. 🤷   It also makes it easy to suppress a message ie. `LOG_LEVEL=NONE crystal spec`

Reply:
Oh, I see what you're saying. I guess it would be nice to be able to tone down the output a little. The only thing is, since this spec actually builds a full Lucky app, you're still stuck with all of the rest of the logging.

Like in this example, the yellow log would be hidden, but the rest would still be there.
<img width="722" alt="Screen Shot 2021-05-17 at 2 16 27 PM" src="https://user-images.githubusercontent.com/2391/118557904-b80fad80-b71a-11eb-8871-17df1aa9f235.png">

Reply:
this might be a larger task but I would not mind applying the same methodology to output there too.

Reply:
I actually looked in to what that would take. It seems we can use `yarn install --silent` and `shards install -q` to tone it down, but it doesn't make it 100% silent. It would take quite a bit more work. Then we also have to figure out how to make that conditional (probably just ENV).